src/Codeception/Lib/Connector/Symfony.php

```diff
@@ -10,11 +10,26 @@ class Symfony extends \Symfony\Component\HttpKernel\Client
     private static $hasPerformedRequest;
 
     /**
+     * @var \Codeception\Module\Symfony
+     */
+    private $module;
+
+    /**
      * @var array
      */
     public $persistentServices = [];
 
     /**
+     * Symfony constructor.
+     * @param \Codeception\Module\Symfony $module
+     */
+    public function __construct($module)
+    {
+        $this->module = $module;
```

Comment:
If connector must create new kernels, connector must be able to set new kernel on symfony module.
It's not mandatory for this branch, but it's a step forward to allow this behaviour.